src/native/external/libunwind_extras/CMakeLists.txt

```diff
@@ -145,13 +130,13 @@ if(CLR_CMAKE_HOST_WIN32)
     add_compile_options(/permissive-)
 
     # include paths
-    include_directories(include/tdep)
-    include_directories(include)
-    include_directories(${CMAKE_CURRENT_BINARY_DIR}/include/tdep)
-    include_directories(${CMAKE_CURRENT_BINARY_DIR}/include)
+    include_directories(${CLR_SRC_NATIVE_DIR}/external/libunwind/include/tdep)
```

Comment:
This file is missing the licensing header.

Reply:
Same as above

Reply:
Ah, you are right, I have incorrectly thought we have it in these files.